.github/workflows/bump-version.yml

```
@@ -1,30 +1,47 @@
 on:
-  pull_request:
-    branches:
-      - main
-    types: [closed]
+    pull_request:
+      branches:
+        - main
+      types: [closed]
+  
 jobs:
-  bump-version:
-    if: github.event.pull_request.merged == true
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          token: ${{ secrets.GH_PUSH_TOKEN }}
-      - name: configure git
-        run: |
-          git remote set-url origin https://itsalaidbacklife@github.com/cuttle-cards/cuttle
-          git config --global user.name "Cuttle Continuous Integration"
-          git config --global user.email "itsalaidbacklife@gmail.com"
-      - name: bump patch version
-        if: contains(github.event.pull_request.labels.*.name, 'version-patch')
-        run: npm run version:patch
-      - name: bump minor version
-        if: contains(github.event.pull_request.labels.*.name, 'version-minor')
-        run: npm run version:minor
-      - name: bump major version
-        if: contains(github.event.pull_request.labels.*.name, 'version-major')
-        run: npm run version:major
-      - name: push
-        run: git push origin main
+    bump-version:
+      runs-on: ubuntu-latest
+  
+      env:
+        GH_PUSH_TOKEN: ${{ secrets.GH_PUSH_TOKEN }}
```

Because the failure itself is the checkout action if the token is missing, I think we might be able to simplify this process by simply coalesce the secret into the default github.token that way it should always be defined
```suggestion
        GH_PUSH_TOKEN: ${{ secrets.GH_PUSH_TOKEN || github.token }}
```

This should ensure we can always check out and therefore skip the need for the additional ifs